### PR TITLE
ZENKO-2961 replication

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

File: src/react/workflow/replication/Replication.jsx [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+                                    <ErrorInput hasError={errors.destinationLocation?.value}> {errors.destinationLocation?.value?.message} </ErrorInput>
+                                </T.ErrorContainer>
+                            </T.Value>
+                        </T.Row>
+                    </T.GroupContent>
+                </T.Group>
+            </T.Groups>
+            <T.Footer>
+                <Button disabled={!isCreating && !showEditWorkflowNotification} id='cancel-workflow-btn' style={{ marginRight: '8px' }} outlined onClick={handleCancel} text='Cancel'/>
+                <Button disabled={!isCreating && !showEditWorkflowNotification} icon={<i className="fas fa-save" />} id='create-workflow-btn' variant="secondary" onClick={handleSubmit(onSubmit)} text={isCreating ? 'Create' : 'Save'}/>
```
Comment (terrydervaux) [inline src/react/workflow/replication/Replication.jsx]: According to the design:
```suggestion
                <Button disabled={!isCreating && !showEditWorkflowNotification} icon={<i className="fas fa-save" />} id='create-workflow-btn' variant="secondary" onClick={handleSubmit(onSubmit)} text={isCreating ? 'Create' : 'Save Changes'}/>
```

File: src/react/Routes.jsx [hunk 2]
```diff
@@ -42,6 +43,10 @@ function Routes() {
                             selected: !!matchPath(pathname, { path: '/buckets' }) ||
                                       !!matchPath(pathname, { path: '/create-bucket' }),
                         },
+                        {
+                            link: <Link to="/workflows">Workflows</Link>,
```
Comment (terrydervaux) [inline src/react/Routes.jsx]: According to the design:
```suggestion
                            link: <Link to="/workflows">Workflow</Link>,
```

File: src/react/workflow/replication/Replication.jsx [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+        return <NoLocationWarning/>;
+    }
+
+    return (
+        <ReplicationContainer>
+            <T.Groups>
+                <T.Group>
+                    <T.GroupContent>
+                        <T.Row>
+                            <T.Key> Rule type </T.Key>
```
Comment (terrydervaux) [inline src/react/workflow/replication/Replication.jsx]: According to the design, `Replication Type` should be in bold white like `General`, `Source` and `Filter`.
Screenshot 2021-03-04 at 12 02 46

Comment (terrydervaux) [inline src/react/workflow/replication/Replication.jsx]:
```suggestion
                            <T.Key> Rule Type </T.Key>
```

File: src/react/ui-elements/ListLayout3.jsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    background-color: ${props => props.theme.brand.primary};
+    min-width: 500px;
+    margin-left: 10px;
+    padding: 10px;
+    border-radius: 5px;
+`;
+
+export const ContentSection = styled.div`
+    display: flex;
+    flex: 0 0 40%;
```
Comment (terrydervaux) [inline src/react/ui-elements/ListLayout3.jsx]: According to the design:
```suggestion
    flex: 0 0 50%;
```

File: src/react/ui-elements/ListLayout3.jsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    flex: 1;
+    flex-direction: row;
+    width: 100%;
+
+    margin: 10px 0px;
+`;
+
+export const ListSection = styled.div`
+    display: flex;
+    flex: 1 1 60%;
```
Comment (terrydervaux) [inline src/react/ui-elements/ListLayout3.jsx]: According to the design
```suggestion
    flex: 1 1 50%;
```

**Comment by Cuervino** [comment]
Logic is good as-is, so those might be in a polishing issue/pr:

- Change the string to "Create New Rule" (with the correct Pascal Case)
- More space between the header "Create New Rule" and "Rule Type" (24px)
- Add "All * are mandatory fields" between this title and "Rule Type", and add a "*" next to all the required fields ("Rule name", Source"Bucket Name", Destination"BucketName")
- Add an icon next to the rule type for instance `<i className="fas fa-coins" />` (see in core-ui https://scality.github.io/core-ui/?path=/story/style-icons--page)
- "Rule Name" instead of "Name"
- States should be with the first letter in Uppercase (Active / Inactive), same in the table.
- Workflow type should be with the first letter in Uppercase in the table (here "Replication" instead of "replication")
- Add the tooltips nest to the label that needs explanation (+ Terry to provide the content)
- 24px spacing between buttons "Cancel" and "Create"

**Review by Cuervino — APPROVED** [review]
Logic is good as-is, so those might be in a polishing issue/pr:

 -   Change the string to "Create New Rule" (with the correct Pascal Case)
 -   More space between the header "Create New Rule" and "Rule Type" (24px)
 -   Add "All * are mandatory fields" between this title and "Rule Type", and add a "*" next to all the required fields ("Rule name", Source"Bucket Name", Destination"BucketName")
-    Add an icon next to the rule type for instance `<i className="fas fa-coins" />` (see in core-ui https://scality.github.io/core-ui/?path=/story/style-icons--page)
-    "Rule Name" instead of "Name"
-    States should be with the first letter in Uppercase (Active / Inactive), same in the table.
-    Workflow type should be with the first letter in Uppercase in the table (here "Replication" instead of "replication")
-    Add the tooltips nest to the label that needs explanation (+ Terry to provide the content)
-   24px spacing between buttons "Cancel" and "Create"

**Review by terrydervaux — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e approve

**Pull request merged**